packages/cli/test/integration/mfa/mfa.api.test.ts

```diff
@@ -150,18 +150,6 @@ describe('Disable MFA setup', () => {
 });
 
 describe('Change password with MFA enabled', () => {
-	test('PATCH /me/password should fail due to missing MFA token', async () => {
-		const { user, rawPassword } = await createUserWithMfaEnabled();
-
-		const newPassword = randomPassword();
-
-		await testServer
-			.authAgentFor(user)
-			.patch('/me/password')
-			.send({ currentPassword: rawPassword, newPassword })
-			.expect(400);
-	});
-
```

Comment:
@RicardoE105 Can you please confirm if this test was invalid?
The `400` is coming from the new password not passing the password policy, and I don't see any MFA checks in any of the `/me` endpoints.

Reply:
Just saw this message. Having a look

Reply:
Looks like this was mistake. But this raises the question: Should we ask for MFA token if the user tries to update the password when it's logged in? 🤔

Reply:
> Should we ask for MFA token if the user tries to update the password

Yes. [we have a ticket for that](https://linear.app/n8n/issue/SEC-12).